relay-config/src/config.rs

```diff
@@ -1014,6 +1014,18 @@ pub struct Processing {
     /// Maximum rate limit to report to clients.
     #[serde(default = "default_max_rate_limit")]
     pub max_rate_limit: Option<u32>,
+    /// True if normalization shouldn't run for events coming from internal relays. Defaults to `false`.
+    #[serde(default)]
+    pub disable_normalization: bool,
+    /// Force relay to run full normalization. Defaults to `false`.
+    ///
+    /// Full normalization includes additional steps that break future
+    /// compatibility and should only run in the last layer of relays.
+    /// Processing relays run it by default, even if the flag is disabled.
+    ///
+    /// This config has no effect if normalization doesn't run.
+    #[serde(default)]
+    pub full_normalize: bool,
```

Comment:
Should we have a single enum instead of two boolean flags? With values `Default`, `Full`, and `Skip`. Because the combination `(True, True)` does not make sense.